tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1212,17 +1212,17 @@ switch (foo) {
 
 switch (foo) {
   case "bar": //comment
-    doThing(); //comment
-
+    doThing();
+   //comment
```

Comment:
Hmm, looks like it's not going to work here. I need to do it on the Switch itself.